### PR TITLE
DOCS-12432 Oct release

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-12432 Oct release- advanced links & credentials population

### CODE REVIEW

**Review by ferferrarazzo — DISMISSED** [review]

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 1]
```diff
@@ -22,11 +22,21 @@ You can specify values for the following parameters of the API Community Manager
 *Result Page URL*:: Specifies the URL of the search results page shown after the user enters a search.
 *Search Areas*:: Selects which types of content to search: *APIs*, *API Groups*, *Applications*, *Knowledge Articles*, *Cases*, *Discussions*, *Topics*, *Managed Content (CMS)*, *API Specifications*, *Documentation Pages*, and *Custom Fields*.
 *Number of results to show*:: Specifies the number of results shown as the user types a query.
+*Enable advance links*:: Enables search results to link to specific tabs of an API details page identified by tab set IDs, such as the console, documentation, custom fields, and applications tabs.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: Is knowing about the tab set IDs important? In other words, would this be just as useful:
```suggestion
*Enable advance links*:: Enables search results to link to specific tabs of an API details page, such as the console, documentation, custom fields, and applications tabs.
```

Comment (kbostrom) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Oh...looking onward, I guess we do have to talk about tab set IDs.
I wonder if there is a less wordy version of "...specific tabs of an API details page identified by tab set IDs."
Ditto in Line 56.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 2]
```diff
@@ -22,11 +22,21 @@ You can specify values for the following parameters of the API Community Manager
...
 
 The *Result Page URL* parameter specifies the URL of a search results page, and the search results page must include an *ACM Search Results* component to display the search results. If the page does not include an *ACM Search Results* component, members and guest users who enter a search and go to the page see no results.
 
 If a content type is not selected in the *Search Areas* parameter, then the linked *ACM Search Results* component omits results of that content type. MuleSoft Government Cloud does not support searching in API specifications or documentation pages.
 
+When you select *Enable advance links*, configure the tab set ID fields that match your customization of the tabs of the API details page, such as `Console Tab Set Id`, `Documentation Tab Set Id`, `Custom fields Tab Set Id`, and `Applications Tab Set Id`:
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:31]: Is `fields` lowercase in the UI label?
Ditto in Line 58.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 3]
```diff
@@ -22,11 +22,21 @@ You can specify values for the following parameters of the API Community Manager
...
+
+. Open the builder.
+. Preview the community as an authenticated user.
+. Open the API details page for an API.
+. Select a tab.
+. In the URL find a tab set ID such as `tabset-1a2b3c=2` or `tabset-1a2b3c=4d5e6f`.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]:
```suggestion
. In the URL, find a tab set ID such as `tabset-1a2b3c=2` or `tabset-1a2b3c=4d5e6f`.
```
"such as" = "similar to"?
Should we identify WHAT URL?

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 4]
```diff
@@ -43,6 +53,16 @@ You can specify values for the following parameters of the Search Results:
 *Component Title*:: Specifies the title for the results.
 *Border*:: Specifies border thickness. Values are *Thick*, *Medium*, *Thin*, or *None*. Default is *Thin*.
 *Show Search Input*:: Shows the search input on the results page.
+*Enable advance links*:: Enables search results to link to specific tabs of an API details page identified by tab set IDs, such as the console, documentation, custom fields, and applications tabs.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: Ditto prev

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 5]
```diff
@@ -43,6 +53,16 @@ You can specify values for the following parameters of the Search Results:
...
+
+When you select *Enable advance links*, configure the tab set ID fields that match your customization of the tabs of the API details page, such as `Console Tab Set Id`, `Documentation Tab Set Id`, `Custom fields Tab Set Id`, and `Applications Tab Set Id`:
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:58]: Ditto prev.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 6]
```diff
@@ -43,6 +53,16 @@ You can specify values for the following parameters of the Search Results:
...
+
+. Open the builder.
+. Preview the community as an authenticated user.
+. Open the API details page for an API.
+. Select a tab.
+. In the URL find a tab set ID such as `tabset-1a2b3c=2` or `tabset-1a2b3c=4d5e6f`.
+. Copy this tab set ID to a tab set ID field such as `Console Tab Set Id`.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:65]: Ditto prev.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 7]
```diff
@@ -303,6 +323,30 @@ You can specify values for the following parameters of the API Console Request P
 *Headline Font Family (Optional)*:: Specifies the headline font family.
 *Error Color (Optional)*:: Specifies the color of error messages.
 
+To enable users to select an application to populate the `clientId` and `clientSecret` credentials, add credential annotations to the application specification:
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: "credentials" = "credential fields"? "credential values"?

**Review by kbostrom — DISMISSED** [review]
CE/CU

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Updated with better wording.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:31]: Yes, the UI label has "fields", although I created a Jira ticket to fix that in the upcoming sprint, and will correct it here when the fix is made.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 3 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Updated to identify what URL and use "similar to", here and on line 64.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 4 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Fixed.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 5 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:58]: Fixed.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 6 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc:65]: Fixed.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 7 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Good catch. Updated to "credential values".

**Review by kbostrom — APPROVED** [review]
CE RA

**Pull request merged**